.github/workflows/build-common.yml

```diff
@@ -290,6 +290,25 @@ jobs:
         if: ${{ !cancelled() && hashFiles('build-scan.txt') != '' }}
         run: cat build-scan.txt
 
+      - name: Get current job url
+        uses: Tiryoh/gha-jobid-action@be260d8673c9211a84cdcf37794ebd654ba81eef # v1.4.0
```

Comment:
I have security worries about using unpopular external actions, is there another way? would it be ok to just link to the run? `https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID`

Reply:
Replaced that action with a different solution that uses `actions/github-script`